airgun/entities/rhai/inventory.py

```diff
@@ -0,0 +1,39 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.rhai import InventoryAllHosts, InventoryHostDetails
+
+
+class InventoryHostEntity(BaseEntity):
+
+    @property
+    def total_systems(self):
+        view = self.navigate_to(self, "All")
+        return view.systems_count.text.split()[0]
+
+    def export_csv(self):
```

Comment:
I believe it will open non-browser dialog as all export actions in application do.  It is not possible to automate such dialogs, so it is better to delete such method completely